cddl/examples/comid-domain-mem.diag

```diff
@@ -9,7 +9,16 @@
   ],
   / triples / 4 : {
     / membership-triples / 5 : [
-      [ / domain - text / "XYZ_Root-of-trust",
+      [
```

Comment:
```suggestion
      [ / Allow generalizing the model to just the vendor /
```

Reply:
Discussed in the meeting with Dionna and when the Domain Definition text was read  in totality the concept was well understood hence resolving the comments!